packages/wallets/src/features/cfd/modals/DxtradeEnterPasswordModal/DxtradeEnterPasswordModal.tsx

```diff
@@ -37,9 +37,9 @@ const DxtradeEnterPasswordModal = () => {
 
     const successDescription = useMemo(() => {
         return accountType === 'demo'
-            ? "Let's practise trading with 10,000 USD virtual funds."
+            ? `Let's practise trading with ${activeWallet?.display_balance} virtual funds.`
```

Comment:
wondering if its time to start adding tests to new functionalities as a standard, anyway, looks good :D

Reply:
I think we can hand that over to the Wallets team 😃 👍🏼